prow/scripts/OWNERS

```diff
@@ -0,0 +1,10 @@
+filters:
+  ".*":
+    labels:
+      - prow/scripts
+
+reviewers:
+  - tooling-reviewers
+
+approvers:
+  - tooling-approvers
```

Comment:
Please add other groups to approvers and reviewers. This should not be responsibility of tooling team only.